dcat/index.html

```diff
@@ -2754,20 +2754,20 @@ <h4>Property: geometry</h4>
             </aside>
 
             <table class="definition">
-              <thead><tr><th>RDF Property:</th><th><a data-cite="LOCN#geometry" title="http://www.w3.org/ns/locn#geometry">locn:geometry</a></th></tr></thead>
+              <thead><tr><th>RDF Property:</th><th><a data-cite="LOCN#locn:geometry" title="http://www.w3.org/ns/locn#geometry">locn:geometry</a></th></tr></thead>
               <tbody>
               <tr><td class="prop">Definition:</td><td>Associates any resource with the corresponding geometry. [[LOCN]]</td></tr>
-              <tr><td class="prop">Range:</td><td><a data-cite="RDF-SCHEMA#ch_literal"><code title="http://www.w3.org/2000/01/rdf-schema#Literal">rdfs:Literal</code></a></td></tr>
-              <tr><td class="prop">Usage note:</td><td>The range of this property is intentionally generic, with the purpose of allowing different geometry encodings. E.g., the geometry could be encoded as <abbr title="Well-known Text">WKT</abbr> (<a href="http://www.opengis.net/ont/geosparql#wktLiteral"><code title="http://www.opengis.net/ont/geosparql#wktLiteral">geosparql:wktLiteral</code></a> [[GeoSPARQL]]).</td></tr>
+              <tr><td class="prop">Range:</td><td><a data-cite="LOCN#locn:Geometry"><code title="https://www.w3.org/ns/locn#Geometry">locn:Geometry</code></a></td></tr>
+              <tr><td class="prop">Usage note:</td><td>The range of this property (<code title="https://www.w3.org/ns/locn#Geometry">locn:Geometry</code>) allows for any type of geometry specification. E.g., the geometry could be encoded by a literal, as <abbr title="Well-known Text">WKT</abbr> (<a href="http://www.opengis.net/ont/geosparql#wktLiteral"><code title="http://www.opengis.net/ont/geosparql#wktLiteral">geosparql:wktLiteral</code></a> [[GeoSPARQL]]), or represented by a class, as <code>geosparql:Geometry</code> (or any of its subclasses) [[GeoSPARQL]].</td></tr>
```

Comment:
Indeed, according to the [LOCN RDF implementation](https://www.w3.org/ns/locn.ttl) the range of `locn:geometry` is  `locn:Geometry`. However, `locn:Geometry is an `rdfs:Class` so I don't think the object of a `lcon:geometry` property can be a literal. .

Reply:
Checking the definition of http://www.opengis.net/ont/geosparql#wktLiteral,  wktLiteral is a `rdfs:Datatype`.

[ `rdfs:Datatype`](https://www.w3.org/TR/rdf-schema/#ch_datatype) is defined in RDF 1.1 as the class of datatypes. All instances of `rdfs:Datatype` correspond to the RDF model of a datatype described in the RDF Concepts specification [RDF11-CONCEPTS]. `rdfs:Datatype` is both an instance of and a subclass of `rdfs:Class`. Each instance of `rdfs:Datatype` is a subclass of `rdfs:Literal`.

@dr-shorthair perhaps this helps to explain why the property can have literals?

Reply:
@dr-shorthair , @riccardoAlbertoni , actually, `locn:Geometry` is meant to be a kind of an "abstract" class / placeholder for any type of geometry representation and encoding (classes or literals), as said in its usage note (https://www.w3.org/ns/locn#locn:Geometry):

> This class defines the notion of "geometry" at the conceptual level, and it shall be encoded by using different formats (see usage note of the locn:geometry property).

The usage note of `locn:geometry` (https://www.w3.org/ns/locn#locn:geometry) says:

> Depending on how a geometry is encoded, the range of this property may be one of the following:
> - a literal (e.g., WKT - string literal -, GML, KML - XML literal)
> - a geometry class, as those defined in the OGC's GeoSPARQL specification, in the W3C's Basic Geo (WGS84 lat/long) vocabulary, and at schema.org;
> - geocoded URIs, as geo or GeoHash URIs, treated as URI references.

Reply:
@dr-shorthair , @riccardoAlbertoni , can we mark this comment as resolved?

Reply:
OK - I understand. These mixed types can cause problems with OWL processors though.

Reply:
@andrea-perego I am in favor of this PR. Only I'd suggest merging #1306 in this PR,  to update the RDF  as well. Being the PR that responds to an erratum can be handy to have a unique pull request to point to in the erratum.

Reply:
Just merged PR https://github.com/w3c/dxwg/pull/1306